testing/ncids-css-testing/stories/components/usa-list/src/usa-list.uswds.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { TestCase } from '../../../../components/test-case';
+import Component from '@nciocpl/ncids-css/internal-uswds/usa-list/src/usa-list.twig';
+import css from './index.scss';
+
+export default {
+	title: 'Components/usa-list/Test Cases',
+};
+
+const Template = (args) => Component(args);
+
+export const USWDSList = () => <TestCase css={css} html={Template.bind({})()} />;
```

Comment:
We don't need a USWDS test case. That's really only for USWDS HTML with NCIDS JS. Remove this and the scenarios.